### PR TITLE
Removed report for others popup screen

### PR DESCRIPTION
**Pull request by bionara**
# Description

Hopefully I didn't get the wrong end of the stick on this @selvach  - I've removed the "report for others" interstitial and code entirely. We may want to boost the prominence of the "report for others" on the thank you page as a result? (see screenshot)

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

image
## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevant
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

**Review by aikewoody — APPROVED** [review]
The removal looks clean but I lack a bit of background information to fully understand the move from "multiple profile bug" to "remove the whole report for others functionality".

File: src/core/profile/ProfileService.ts [hunk 1]
```diff
@@ -1,7 +1,5 @@
-import { AsyncStorageService } from '@covid/core/AsyncStorageService';
 import { IPatientService } from '@covid/core/patient/PatientService';
 import { Services } from '@covid/provider/services.types';
-import { cleanIntegerVal } from '@covid/utils/number';
 import { inject, injectable } from 'inversify';
 
 const MAX_DISPLAY_REPORT_FOR_OTHER_PROMPT = 3;
```
Comment (aikewoody) [inline src/core/profile/ProfileService.ts:5]: This const is not used anymore

Comment (bionara) [reply src/core/profile/ProfileService.ts]: deleted, thanks!

**Comment by selvach** [comment]
Looks good Ben! So nice to delete some code :)
I do not think think there is anything to clear up on the backend?

**Review by selvach — APPROVED** [review]

**Pull request merged**